src/Word.jl

```diff
@@ -0,0 +1,68 @@
+abstract type Word{T} <: AbstractVector{T} end
+
+
+
+#interface for morphism T->Int
+function to_int{T}(t::T) end
```

Comment:
in julia there is usually no need to specify those functions. Well written libraries should handle any type of `Integers`

Reply:
and this is an old syntax (pre julia-1.0), now it should be written as
```julia
function to_int(t::T) where T
    ...
end
```
or as a one-liner:
```julia
to_int(t::T) where T = ...
```